.gitignore

```diff
@@ -31,8 +31,8 @@ bower_components
 build/Release
 
 # Dependency directories
-node_modules/
-jspm_packages/
+node_modules
+jspm_packages
```

Comment:
The pattern `node_modules` matches either a file or folder, whereas `node_modules/` matches a folder.

As a general convention, this practice therefore seems wrong:
1. It accidentally matches files which is slightly less efficient
2. For human readers of this file, it no longer clearly communicates that we expect `node_modules` to be a folder not a file

Reply:
This comes from the `rush init` template, so we should fix it there.